archetypes/blog.md

```diff
@@ -0,0 +1,24 @@
+---
+type: blog # So lassen
+title: Mein neuer Blogpost
+author: Oliver OpenData  # wenn nicht vorhanden steht da dann codeforde
+twitter_handle: # wenn nicht vorhanden steht da dann @codeforde
+excerpt: Ein bis zwei Sätze die den Blogpost beschreiben. Wird für die Blogliste verwendet.
+topic: event # zb event, community, default, interview, project, story, toolbox
+images:
+   - imgname: bild.jpg # Dieses Bild sollte im Verzeichnis static/blog existieren
+
+og_image: header.img # Der Dateiname vom Titelbild 
```

Comment:
Viellicht hier noch erwähnen, wo das Bild liegen muss. Ist das auch in `/static/blog`?